styles/base/form.scss

```diff
@@ -93,28 +80,25 @@ label {
     z-index: 2;
     float: left;
     width: 100%;
+    border-right: 0;
+    border-left: 0;
+
+    &:focus { 
+      box-shadow: 0 0 2px 1px rgba(82,181,229,1);
```

Comment:
Yea, and it's well-appreciated. But simple changes like this (regarding the shadows) needs a bit of approval because it doesn't mesh well with our components (it can cause inconsistencies).

If  you want changes like this, put it on a separate PR, not mixing it with another one.